jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py

```diff
@@ -69,6 +71,43 @@ def __init__(self):
             self.gchat_image_dir = rospy.get_param("~google_chat_tmp_image_dir", "/tmp")
             self._gchat_thread = None
 
+    def _reconfigure_cb(self, config, level):
+        self.use_mail = config['use_mail']
+        self.use_twitter = config['use_twitter']
+        self.use_google_chat = config['use_google_chat']
+        self.send_every_transition = config['send_every_transition']
+        rospy.loginfo(
+            "Switched parameters; use_mail: {send_every_transition}, "
```

Comment:
Sorry, I made a mistake.
```suggestion
            "Switched parameters; use_mail: {use_mail}, "
```